daml-foundations/daml-ghc/src/DA/Daml/GHC/Compiler/Convert.hs

```diff
@@ -891,6 +877,20 @@ convertExpr env0 e = do
     convertArg env = \case
         Type t -> TyArg <$> convertType env t
         e -> TmArg <$> convertExpr env e
+    withTyArg :: Env -> (LF.TypeVarName, LF.Kind) -> [LArg Var] -> (LF.Type -> [LArg Var] -> ConvertM (LF.Expr, [LArg Var])) -> ConvertM (LF.Expr, [LArg Var])
```

Comment:
I'd rather these functions had a newline before them, and I think I'd even prefer if they were at the top level. They seem generic combinators with general properties.